### PR TITLE
Downgrade cmake for samples

### PR DESCRIPTION
**Pull request by mryzhov**
Downgraded cmake version to default version for Ubuntu 18.04

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: build-instruction.md [hunk 1]
```diff
@@ -452,7 +452,8 @@ The software was validated on:
 
 ### Software Requirements
```
Comment (artanokhov) [inline build-instruction.md:453]: Please update info about Python requirements(Python3.5 -> Python3.6) because of end of support Python3.5

**Review by artanokhov — APPROVED** [review]

File: build-instruction.md [hunk 2]
```diff
@@ -452,7 +452,8 @@ The software was validated on:
...
 
-- [CMake]\* 3.11 or higher
+- [CMake]\* 3.13 or higher for the distribution package 
+- [CMake]\* 3.10 or higher for the samples and demos only
 - Clang\* compiler from Xcode\* 10.1 or higher
 - Python\* 3.5 or higher for the Inference Engine Python API wrapper
```
Comment (andrew-zaytsev) [inline build-instruction.md:459]: Consider adding the following note about CMake version required for Intel Distribution:

> **NOTE**: Building samples and demos from the Intel® Distribution of OpenVINO™ toolkit package requires CMake\* 3.10 or higher.

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: build-instruction.md (hunk 1 above)
Comment (mryzhov) [reply build-instruction.md:455]: thanks, fixed

File: build-instruction.md (hunk 2 above)
Comment (mryzhov) [reply build-instruction.md:461]: Added

**Review by artanokhov — APPROVED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@azhogov please, merge

**Pull request merged**